Replicator/tests/SG.cc

```diff
@@ -166,9 +168,9 @@ void SG::flushDatabase() const {
     runRequest("POST", { }, "_flush", nullslice, true);
 }
 
-bool SG::insertBulkDocs(C4CollectionSpec collectionSpec, const slice docsDict) const {
+bool SG::insertBulkDocs(C4CollectionSpec collectionSpec, const slice docsDict, double timeout) const {
```

Comment:
Have a default value so that you don't need to type in one every time?

Reply:
It's in the `.hh` file